src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -468,6 +468,7 @@ internal override void ForceComplete(SourceLocation locationOpt, CancellationTok
                             var diagnostics = DiagnosticBag.GetInstance();
                             AfterMembersChecks(diagnostics);
                             AddDeclarationDiagnostics(diagnostics);
+                            DeclaringCompilation.SymbolDeclaredEvent(this);
```

Comment:
Please add a comment clearly stating when do we intend to produce this event so that it would be clear why CompletionPart.StartMemberChecks is the right point to do this, why not earlier or later than that.